CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -745,16 +747,29 @@ bool IRenderWebBrowser.StartDragging(IDragData dragData, DragOperationsMask allo
             currentDragData = dragData.Clone();
             currentDragData.ResetFileContents();
 
-            // TODO: The following code block *should* handle images, but GetFileContents is
-            // not yet implemented.
-            //if (dragData.IsFile)
-            //{
-            //    var bmi = new BitmapImage();
-            //    bmi.BeginInit();
-            //    bmi.StreamSource = dragData.GetFileContents();
-            //    bmi.EndInit();
-            //    dataObject.SetImage(bmi);
-            //}
+            if (dragData.HasImage)
+            {
+                IImage dragImage = dragData.Image;
+                int width, height;
+                byte[] pixels = dragImage.GetAsBitmap(1f, ColorType.Rgba8888, AlphaType.PreMultiplied, out width, out height);
+                int stride = ((width * 32 + 31) & ~31) / 8;
```

Comment:
Looks like your using `PixelFormats.Pbgra32` below, so it should be the correct format. Use 

`(PixelFormats.Pbgra32.BitsPerPixel / 8) * width` to calculate `stride`

https://github.com/cefsharp/CefSharp/blob/cefsharp/71/CefSharp.Wpf/Rendering/InteropBitmapRenderHandler.cs#L31